config/routes.rb

```diff
@@ -10,7 +10,9 @@
 
   namespace :admin do
     resources :dashboard, only: [:index]
-    resources :companies, only: [:show, :destroy, :update]
+    resources :companies, only: [:show, :destroy, :update] do
+      resources :notes, only: [:index, :create, :destroy]
+    end
     resources :pdfs, only: [:create, :new]
     resource :tools_resources, only: [:edit]
   end
```

Comment:
I agree, As we discussed this morning, I think a notes controller is great, this gives us the ability for each indvidual admin to add notes, and the other admins can see who added the note.